aws/dynamodb_test.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func CreateTestDynamoTables(t *testing.T, session *session.Session) error {
+	svc := dynamodb.New(session)
+	tablestocreate := []string{"Table1", "Table2"}
```

Comment:
These table names need to include a unique, randomly generated ID to avoid conflicts when there are many tests running in parallel. See some of the other tests for the `UniqueId()` function we use for this.